docs/frameworks/sveltekit.md

```diff
@@ -0,0 +1,215 @@
+---
+title: SvelteKit | Frameworks
+---
+
+# SvelteKit
+
+> For `Type declarations`, `Prompt for update` and `Periodic SW Updates` go to [Svelte](/frameworks/svelte.html) entry.
+> 
+
+You should remove all references to [SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker) <outbound-link /> to disable it on your application.
+
+Since `SvelteKit` uses `SSR / SSG`, we need to add the `ReloadPrompt` component using `dynamic import`. `Vite Plugin PWA` will only register the service worker on build, it is aligned with the current behavior of [SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker) <outbound-link />.
+
+The best place to include the `ReloadPrompt` is on the main layout of the application:
+
+<details>
+  <summary><strong>src/routes/__layout.svelte</strong> code</summary>
+
+```html
+<script>
+    import { onMount } from 'svelte'
+    import { browser, dev } from '$app/env'
+
+    let ReloadPrompt
+    onMount(async () => {
+      !dev && browser && (ReloadPrompt = (await import('$lib/components/ReloadPrompt.svelte')).default)
+    })
+</script>
+
+<svelte:head>
+    {#if (!dev && browser)}
+      <link rel="manifest" href="/_app/manifest.webmanifest">
+    {/if}
+</svelte:head>
+
+<main>
+  <slot />
+</main>
+
+{#if ReloadPrompt}
+  <svelte:component this={ReloadPrompt} />
+{/if}
+```
+</details>
+
+## SvelteKit Adapters
+
+The main problem with the current implementation of the service worker module of `SvelteKit` is that you don't have access to the result applied by any adapter you have configured on your application.
+
+The service worker module of `SvelteKit` will be called before the adapter logic is applied, and so, inside the service worker module, you don't have access to those resources.
+
+Your application will not work when the user is offline, since the pages will not be included on the service worker precache manifest.
```

Comment:
These sentences all seem related and so should probably be part of the same paragraph
```suggestion
The main problem with the current implementation of the service worker module of `SvelteKit` is that you don't have access to the result applied by any adapter you have configured on your application. The service worker module of `SvelteKit` will be called before the adapter logic is applied, and so, inside the service worker module, you don't have access to those resources. Your application will not work when the user is offline, since the pages will not be included on the service worker precache manifest.
```